### PR TITLE
Add E2E job which build and deploy AnsibleEE

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-ansibleee-operator-main-ansibleee-operator-build-deploy

File: ci-operator/config/openstack-k8s-operators/openstack-ansibleee-operator/openstack-k8s-operators-openstack-ansibleee-operator-main.yaml [hunk 1]
```diff
@@ -1,22 +1,39 @@
...
+  cli:
+    name: "4.10"
+    namespace: ocp
+    tag: cli
   golangci-lint:
     name: golangci-lint
     namespace: ci
     tag: v1.46.0
-binary_build_commands: make build
+  openstack-operator-ci-pre-commit:
```
Comment (sandeepyadav93) [inline ci-operator/config/openstack-k8s-operators/openstack-ansibleee-operator/openstack-k8s-operators-openstack-ansibleee-operator-main.yaml:10]: We are moving to use the same build_root image for pre-commit job as well.

For some operators we have already done that, could you please follow the example from mariadb operator:

[1] https://github.com/openshift/release/blob/master/ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master.yaml#L33-L37
[2] https://github.com/openshift/release/blob/master/ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master.yaml#L93-L103


fyi.. there are open patches to do same for other operators:

https://github.com/openshift/release/pulls?q=is%3Apr+is%3Aopen+amol

**Review by sandeepyadav93 — CHANGES_REQUESTED** [review]

**Comment by fao89** [comment]
@sandeepyadav93 any clue why it is failing now?

**Comment by fao89** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-ansibleee-operator-main-openstack-ansibleee-operator-build-deploy

**Pull request merged**